Orange/preprocess/preprocess.py

```diff
@@ -569,6 +570,32 @@ def __call__(self, data):
             data = pp(data)
         return data
 
+class SelectDense(Preprocess):
+    """
+    Select dense features. Denseness is determined according to
+    user-defined treshold.
+
+    Parameters
+    ----------
+    treshold : float
+        Minimal proportion of non-zero entries of a feature
+    """
+
+    def __init__(self, treshold=0.05):
+        self.treshold = treshold
```

Comment:
I don't want to be this person, but perhaps for clarity and for maintaining a good API, I would correct the name to _threshold_.

Reply:
Thanks. This, along with length is in my top misspelled words.